docs/formats/zst.md

```diff
@@ -270,11 +270,21 @@ An `<array_column>` has the form:
 where
 * `values` represents a continuous sequence of values of the array elements  
 that are sliced into array values based on the length information, and
-* `lengths` encodes a Zed int32 sequence of values that represent the length
+* `lengths` encodes a Zed `int32` sequence of values that represent the length
  of each array value.
 
 The `<array_column>` structure is used for both Zed arrays and sets.
 
+#### Map Column
+
+A `<map_column>` has the form:
+```
+{key:<any_column>,value:<any_column>}
+```
+where
+* `key` encodes the column of map keys, and
+* `value` encodes the column of map values.
+
```

Comment:
I moved this section above the one for "union" because @nwt conditioned me to make Zed-related complex type docs to follow the order in [https://github.com/brimdata/zed/blob/main/type.go](https://github.com/brimdata/zed/blob/main/type.go).